docs/source/function_description.rst

```diff
@@ -0,0 +1,22 @@
+Function description
+====================
+
+The functions of our pipeline
+
+----
+
+.. module:: src
+
+.. autofunction:: initPTB()
+
+:func:`initPTB`
```

Comment:
I don't understand this part, what is it the point to have `:func:`initPTB`` and `.. autofunction:: getExperimentStart` after it?

Reply:
```suggestion
```
You are right: this one can be removed and autofunction getExpStart brought up. :-)